logicrunner/executionbroker.go

```diff
@@ -769,6 +578,93 @@ func (q *ExecutionBroker) NoMoreRequestsOnLedger(ctx context.Context) {
 	q.stopRequestsFetcher(ctx)
 }
 
+func (q *ExecutionBroker) PendingState() insolar.PendingState {
```

Comment:
it is a public method, i'm afraid that we catch deadlock on q.stateLock.

Reply:
I don't understand this concern. All methods that I put in the interface are "thread save", the rest should go private.